src/common/lib/keylayers.c

```
@@ -388,18 +297,61 @@ static void keylayers_handle_osl(uint8_t target_layer, bool make) {
- * Handles MO (momentary), TG (toggle), TO (switch to), and OSL (one-shot) operations.
- * Updates layer_state based on keycode and make/break.
- *
- * Implementation uses bitmap operations and array indexing.
- * Config persistence triggered only for TG and TO (not MO or OSL).
- *
- * @param code The layer operation keycode (0xF0-0xFF)
- * @param make true if key pressed, false if released
- */
+// --- Public Functions ---
+
+void keylayers_reset(void) {
```

Cppcheck (2.19.0)

[style] The function 'keylayers_reset' is never used.

(unusedFunction)